src/library-authoring/data/apiHook.js

```diff
@@ -17,6 +30,22 @@ export const useContentLibrary = (libraryId) => (
   })
 );
 
+/**
+ * Use this mutation to create a block in a library
+ * @param {string} [libraryId]
+ */
+export const useCreateLibraryBlock = (libraryId) => {
+  const queryClient = useQueryClient();
+  return useMutation({
+    /** @type {import("@tanstack/react-query").MutateFunction<any, any, CreateBlockData>} */
+    mutationFn: async (data) => createLibraryBlock(data),
+    onSettled: () => {
```

Comment:
We could use the `data` parameter of the `onSettled` event to get the `libraryId`. That way we can get rid of the `libraryId` parameter in the hook.